evento/src/views/EventPage/index.js

```diff
@@ -36,20 +54,46 @@ class EventPage extends Component {
 		});
 	}
 
+	onAttendButtonClick() {
+		const eventId = this.props.match.params.eventId;
+
+		fetch(`/events/${eventId}/attendees`, {
+			method: 'POST',
+			headers: { 'Authorization': Cookie.get('auth_token') }
+		})
+		.then(response => {
+			if (!response.ok) return Promise.reject();
+			// If successiful attending, re-fetch attendees
+			this.fetchAttendees();
+		})
+		.catch(() => this.setState({ errorMessage: "Something went wrong" }));
+	}
+
+	isUserAttending() {
+		let user = Cookie.get('user');
+		return JSON.stringify(this.state.attendees).includes(user);
```

Comment:
I definitey prefer the second option. Even though it is longer, it is much more "correct" and less hacky solution.

One way to make the code a bit cleaner is this:

```
const userCookie = Cookie.get('user');
const user = userCookie ? JSON.parse(userCookie) : {};
```

Or probably event better to just check `if(!userCookie) return false;` because otherwise it is comparing to a non-existent .id field